WcaOnRails/app/models/post.rb

```diff
@@ -2,11 +2,37 @@
 
 class Post < ApplicationRecord
   belongs_to :author, class_name: "User"
+  has_many :post_tags, autosave: true, dependent: :destroy
 
   validates :title, presence: true, uniqueness: true
   validates :body, presence: true
   validates :slug, presence: true, uniqueness: true
 
+  def reload
```

Comment:
I started to make this change, but ran into trouble. This actually makes the test more complicated than you might expect. We cannot write `post = Post.find(post.id)` because of the behavior of the `let(:post)` construct (it doesn't allow you to update post). I think keeping the reload method here is simplest.